### ISSUE TITLE
Announce this project to the wider Apache Arrow community

### ISSUE THREAD

**Issue by pitrou** [man-group/sparrow#85]
Do you plan to announce the sparrow project on one of the [Arrow mailing-lists](https://arrow.apache.org/community/)? It could interest other parties and perhaps elicit suggestions from the community.

(apologies if you've already done so and I haven't noticed)

**Comment by pitrou** [comment]
cc @jorisvandenbossche

**Comment by JohanMabille** [comment]
@pitrou sorry for the late reply,

We plan to announce it, but we want to reach a certain level of maturity and have a minimum set of features implemented before doing so. I hope we can do it by next month.

**Comment by SylvainCorlay** [comment]
> We plan to announce it, but we want to reach a certain level of maturity and have a minimum set of features implemented before doing so. I hope we can do it by next month.

Yeah we wanted to reach out to you both personally before making any kind of announcement. Feedback is very welcome, although it is not ready for prime time yet.
